docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,155 @@
+
+```xml
+<property>
+  <name>dt.operator.{OperatorName}.prop.tableName</name>
+  <value>mytable</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.columnsExpression</name>
+  <value>column1,column2,column4</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.key</name>
+  <value>keycolumn</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.databaseDriver</name>
+  <value>com.mysql.jdbc.Driver</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.databaseUrl</name>
+  <value>jdbc:mysql://localhost:3306/mydb</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.connectionProps</name>
+  <value>user:myuser,password:mypassword</value>
+</property>
+```
+
+* If you need to filter the table records, set `whereCondition` which will be added to the generated sql query.
+* If you have huge database to scan set `partitionsCount` to a higher number to increase read parallelism.
+* The operator uses PreparedStatement, a precompiled SQL statement to fetch records from database table. You can set `fetchSize` to hint database driver to restrict number of rows to fetch in one call. The remaining rows of resultset will be fetched in subsequent calls.
+* The platform invokes the emitTuples() callback multiple time in each streaming window; within a single such call, if a large number of tuples are emitted, there is some risk that they may overwhelm the downstream operators especially if they are performing some compute intensive operation. For such cases, output can be throttled by reducing the value of the `batchSize` property. Conversely, if the downstream operators can handle the load, increase the value to enhance throughput.
+* If there is high frequency of incoming records in your table and you want to process it as soon they appear use lower value of `pollInterval`; if they appear rarely or if some delay in processing a new file is acceptable, increase it.
+* After reading the records from table they are held in memory for some time till they are emitted to next operator. The records are kept in a blocking queue. The capacity of this blocking queue can be changed using parameter `queueCapacity`.
+
+
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *store* | JDBC Store for connection | [JDBCStore](#JDBCStore) | Yes | N/A |
+| *tableName* | table name to be scanned | String | Yes | N/A |
+| *columnsExpression* | Comma separated list of columns to select from the given table. | String | Yes | N/A |
+| *key* | Primary key column name | String | Yes | N/A |
+| *partitionCount* | Static partitions count | int | No | 1 |
+| *whereCondition* | Where condition for jdbc query | String | No | N/A |
+| *fetchSize* | The number of rows to fetch by PreparedStatement | int | No | 20000 |
```

Comment:
Hint limiting the  number of rows to fetch in a single call. Also, seems like the default value would be driver dependent. It would be immensely useful to the reader if we can document the default value for a few commonly
used drivers. For example, it looks like the MySQL treats this value in a rather odd way:
 http://stackoverflow.com/questions/20496616/fetchsize-in-resultset-set-to-0-by-default